promgen/urls.py

```diff
@@ -126,6 +126,8 @@
     path("proxy/v1/silences/<silence_id>", csrf_exempt(proxy.ProxyDeleteSilence.as_view()), name="proxy-silence-delete"),
     # Promgen rest API
     path("rest/", include((router.urls, "api"), namespace="api")),
+    # PromQL Query
+    path("promql-query", views.PromqlQuery.as_view(), name="promql-query"),
```

Comment:
This is more of a 'thinking out loud' comment, but we have a few internal APIs, like this, exporter test, and so on, I wonder if there would be value in making some kind of `internal/` prefix for them, just to help organize them.  There's also some older API cleanup I would like to so, so picking some kind of internal prefix might help us see what have been migrated so far.

Reply:
> but we have a few internal APIs, like this, exporter test, and so on, I wonder if there would be value in making some kind of internal/ prefix for them, just to help organize them

We could. But since the others are not already organized under the same `internal/` prefix, I guess we could leave this one as it is, and organize all of them in the future.

Or, if you tell me which `path`s on the `urls.py` file you want organized under an `internal/` prefix, I can do that on this PR.